libcst/matchers/tests/test_matchers_with_metadata.py

```diff
@@ -0,0 +1,473 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# pyre-strict
+from textwrap import dedent
+from typing import Sequence, Set, Tuple
+
+import libcst as cst
+import libcst.matchers as m
+import libcst.metadata as meta
+from libcst.matchers import matches
+from libcst.testing.utils import UnitTest
+
+
+class MatchersMetadataTest(UnitTest):
+
+    ONCALL_SHORTNAME = "instagram_server_framework"
```

Comment:
Remove?

Reply:
Haha whoops. I'll scan the repo and remove those.